tracee-rules/output.go

```diff
@@ -26,8 +32,10 @@ Command: {{ .Context.ProcessName }}
 Hostname: {{ .Context.HostName }}
 `
 
-func setupTemplate(inputTemplateFile string) (*template.Template, error) {
+func setupTemplate(inputTemplateFile string) (exporter, error) {
```

Comment:
Returning interfaces is not ideal as it loosens the abstraction we've made. It also bypasses any guard rails for compile time checks for problems to surface by pushing them into runtime. This will also complicate things when we ever have to change this interface. 

Read more https://github.com/golang/go/wiki/CodeReviewComments#interfaces